packages/sign_in_with_apple/README.md

```diff
@@ -1,9 +1,10 @@
 # Sign in With Apple
 
-Flutter bridge to Sign in with Apple.
-
+Flutter bridge to Sign in with Apple.\
 Supports login via an Apple ID, as well as retrieving credentials saved in the user's keychain.
 
+<img src="https://flutter.dev/assets/development/packages-and-plugins/FlutterFavoriteLogo-c581613beba0abd3a92fa9d1c86e7b38062f92d104347a3a6bb28841233331fd.png" width="100" />
```

Comment:
Do you think this will stay up? Else we can just put it into the Git repo here…

Reply:
In https://github.com/aboutyou/dart_packages/tree/master/assets

Reply:
pub.dev uses https://pub.dev/static/img/ff.svg?hash=e6eqku0t4nirho4jt8akssianh0lvuiv, while `provider` for example sports a copy as well.

Reply:
Done